ui/components/ui/tooltip/tooltip.js

```diff
@@ -82,7 +82,7 @@ export default class Tooltip extends PureComponent {
         style={style}
         title={disabled ? '' : title}
         trigger={trigger}
-        theme={theme}
+        theme={`tippy-tooltip--mm-custom ${theme}`} // Required for correct theming
         tabIndex={tabIndex || 0}
         tag={tag}
       >
```

Comment:
Adding default theme that handles light/dark mode colors but allowing for custom theme functionality if more customization is needed e.g. InfoTooltip max width